docs/source/science/NISAR/OPERA Surface Displacement.ipynb

```diff
@@ -0,0 +1,4440 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    ds = xr.open_dataset("downloaded_file.nc")
</pre><p>You can delete this line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>